src/ddqa/cli/create/__init__.py

```diff
@@ -22,10 +22,27 @@
     multiple=True,
     help='Labels that will be attached to created issues',
 )
+@click.option(
+    '-i',
+    '--include-labels',
+    'include_labels',
```

Comment:
Unnecessary since it is the same as the long version

```suggestion
```

Reply:
Ah that makes sense. On it!